backend/services/videoPromptService.js

```diff
@@ -133,11 +128,12 @@ const createVideoPrompt = async ({ input, ipsId, userId }) => {
             promptSpaceId: ips._id,
             input: {
                 message,
-                fullMessage,
                 samplingSteps,
                 cfgScale,
                 eta,
                 fps,
+                width,
+                height,
             },
```

Comment:
What I mean is you are creating a new Video prompt (mongoose type) with the value above, samplingsteps, cfgscale and more. But they are not defined in the actual mongoose schema.
![image](https://github.com/user-attachments/assets/c95c3885-7456-4a37-97e7-7596b7f1109c)